### DISCUSSION TITLE
关于解码数据最短长度的问题

### DISCUSSION THREAD

**Discussion by F-Y-all** [wenet-e2e/WenetSpeech#26] [Q&A]
前辈们好，有一条测试数据：TEST_NET_Y0000000004_0ub4ZzdHzBc_S00023	TEST_NET_Y0000000004_0ub4ZzdHzBc	180.65	180.73
这个数据只有80ms，经过（窗口大小25ms，窗移10ms）数据处理提出(1,6,80) 维的特征，经过convsubsampling layer的第一个卷积层（size=3，stride=2）之后变成(1,1,2,19)，这个时候再过第二个卷积层(size=3,stride=2)，这时特征size(2)就小于kernel的size(3)然后就报错了。我有两个问题，1. 降采样曾是否需要进行padding？2. 小于85ms的音频还送入网络识别吗（85ms是降采样卷积层不padding不报错的最短音频时长）？

**Comment by robin1001** [comment]
这个我们应该当时直接忽略了，没有做其他处理，你那边直接忽略就行。

**Reply by F-Y-all** [reply]
好的感谢！
